src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -146,13 +148,41 @@ julia> rays(nc)
 2-element SubObjectIterator{RayVector{QQFieldElem}}:
  [1, 0]
  [0, 1]
+
+```
+
+An example of building the normal cone for the first two vertices of the square (in this case this is the edge with vertices [-1,-1] and [1,-1]).
+```jldoctest
+julia> square = cube(2)
+Polytope in ambient dimension 2
+
+julia> C = normal_cone(square, [1,2])
+Polyhedral cone in ambient dimension 2
+
+julia> facets(C)
+1-element SubObjectIterator{LinearHalfspace{QQFieldElem}} over the halfspaces of R^2 described by:
+-x_2 <= 0
+
+julia> C = normal_cone(square, [1,2], outer = true)
+Polyhedral cone in ambient dimension 2
+
+julia> facets(C)
+1-element SubObjectIterator{LinearHalfspace{QQFieldElem}} over the halfspaces of R^2 described by:
+x_2 <= 0
+
 ```
 """
-function normal_cone(P::Polyhedron{T}, i::Int64) where {T<:scalar_types}
-  @req 1 <= i <= n_vertices(P) "Vertex index out of range"
-  bigobject = Polymake.polytope.normal_cone(pm_object(P), Set{Int64}([i - 1]))
+function normal_cone(
+  P::Polyhedron{T}, F::AbstractVector; outer::Bool=false
+) where {T<:scalar_types}
+  @req 1 <= length(F) <= n_vertices(P) "Face index out of range"
+  @req all(1 .<= F .<= n_vertices(P)) "Vertex index out of range"
+  bigobject = Polymake.polytope.normal_cone(
+    pm_object(P), Set{Int64}(F .- 1); outer=Int64(outer)
+  )
   return Cone{T}(bigobject, coefficient_field(P))
 end
+normal_cone(P::Polyhedron{T}, i::Int64) where {T<:scalar_types} = normal_cone(P, [i])
```

Comment:
```suggestion
normal_cone(P::Polyhedron{T}, i::Int64; outer::Bool=false) where {T<:scalar_types} = normal_cone(P, [i]; outer=outer)
```
I think this should get the `outer` option as well, an added keyword argument is fine regarding compatibility with the old version.